internal/handlers/users.go

```diff
@@ -18,70 +15,23 @@ type CreateUserParams struct {
 
 //go:generate moq -rm -out users_mock.go . UsersService
 type UsersService interface {
-	Create(ctx context.Context, name string) (models.User, error)
 	GetOne(ctx context.Context, id string) (models.User, error)
 }
 
 type Users struct {
+	protobuf.UnimplementedUsersServer
 	user UsersService
 }
 
 func NewUsers(us UsersService) Users {
 	return Users{user: us}
 }
 
-func (u Users) Routes() http.Handler {
```

Comment:
instead of replacing http handlers we could create separate package with grpc handlers and make it dependent on `user UsersService` too.

Reply:
Ok - I thought that we should replace it instead of making it an option :).